src/relentless/simulate/initialize.py

```diff
@@ -165,7 +171,7 @@ def _pack_particles(seed, N, V, diameters):
             if len(trees) > 0:
                 mask = numpy.ones(sites.shape[0], dtype=bool)
                 for j, treej in trees.items():
-                    dj = diameters[j]
+                    dj = variable.evaluate(diameters[j])
```

Comment:
This wasn't caught in the unit tests but failed when testing my own script with one diameter value being a design variable.

Reply:
Thank you! Yes, that needs to be evaluated here. Can you modify / add to the unit tests so that this case is covered?

Reply:
Since it now packs by diameters and those diameters stay objects, the test fails since the tree will include type B (the design variable). I don't think the tests need modified now.